dependencies.yaml

```diff
@@ -72,6 +72,7 @@ devel_no_selenium_requires:
   xorg-x11-fonts:
   perl(Devel::Cover):
   perl(Devel::Cover::Report::Codecov):
+  perl(JSON::MaybeXS):
```

Comment:
what is this specifically needed for? Shouldn't it be an implicit dependency of some other package?

Reply:
I believe it should.

Reply:
Unless it's the only missing dependency for the module that cdywan downloads from CPAN?

Reply:
Yes, except *Devel::Cover::Report::Codecovbash* is not a package yet.

Reply:
But since you are installing it via cpanm, the dependency should be installed automatically

Reply:
ok, shouldn't matter. We can keep it as a task for later in the related ticket or a followup to use properly packaged dependencies.

Reply:
For now it makes sense to have the dependency so we need to install as less as possible via cpanm.

Reply:
`JSON::MaybeXS` is a super light depedency. Installing it via `cpanm` should be no problem at all.